qiskit/tools/visualization/_state_visualization.py

```diff
@@ -321,8 +395,25 @@ def phase_to_color_wheel(complex_number):
     return color_map[angle_round]
 
 
-def plot_state_qsphere(rho, filename=None, show=False):
-    """Plot the qsphere representation of a quantum state."""
+def plot_state_qsphere(rho, figsize=None):
```

Comment:
When running this we print the eigenvalues before generating the visualization. Do you think that is still needed? It's a bit weird to have the plot function print to stdout